x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java

```diff
@@ -669,6 +669,70 @@ public void testFailOverOnFollower() throws Exception {
         unfollowIndex("follower-index");
     }
 
+    public void testAddNewReplicasOnFollower() throws Exception {
```

Comment:
I wonder if we should make this less heavy. Maybe one background thread is enough?

Reply:
Yes, one thread is enough. I'll update this.